pagopa/pn-frontend-e2e
# PN-9926-e-2-e-mittente-mittente-scarica-il-file-attestazione-opponibile-a-terzi-notifica-digitale

Issue Description
Some of the tests tagged with `@DownloadFileMittente` have a short time limit, which prevents them from completing the file download process. Furthermore, the feature files do not contain the JIRA ID.

Changes Made:
- added feature step wait time before finding and downloading the file
- added `lombok` dependency into the `pom.xml`

How to Test
Simply run all tests with the tag `@DownloadFileMittente` and check if they work properly.
